continuous_integration/environment-3.8.yaml

```diff
@@ -5,8 +5,8 @@ channels:
 dependencies:
   # required dependencies
   - python=3.8
-  - numpy=1.20
-  - pandas=1.2
+  - numpy
+  - pandas
```

Comment:
We tend to use the different Python version builds to also test against different versions of NumPy and pandas. Could you pin NumPy and pandas for this build to be something between the Python 3.7 and 3.9 builds?

Reply:
I'm planning to shift all versions in the PR where I drop python 3.6